modules/rosa-policy-responsibilities.adoc

```diff
@@ -0,0 +1,55 @@
+:system-module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-policy-responsibility-matrix.adoc
+
+[id="rosa-policy-responsibilities_{context}"]
+= Overview of responsibilities for {product-title}
+
+[role="_abstract"]
+While Red Hat and Amazon Web Services (AWS) manage the {product-title} service, the customer shares responsibility with respect to certain aspects. The {product-title} services are accessed remotely, hosted on public cloud resources, created in customer-owned cloud service provider accounts, and have underlying platform and data security that is owned by Red Hat.
+
+[IMPORTANT]
+====
+If the `cluster-admin` role is enabled on a cluster, please see the responsibilities and exclusion notes in the link:https://www.redhat.com/en/about/agreements[Red Hat Enterprise Agreement Appendix 4 (Online Subscription Services)].
+====
+
+[cols="2a,3a,3a,3a,3a,3a",options="header"]
+|===
+
+|Resource
+|Incident and operations management
+|Change management
+|Identity and access management
+|Security and regulation compliance
+|Disaster recovery
+
+|Customer data |Customer |Customer |Customer |Customer |Customer
+
+|Customer applications |Customer |Customer |Customer |Customer |Customer
+
+|Developer services |Customer |Customer |Customer |Customer |Customer
+
+|Platform monitoring |Redhat |Redhat |Redhat |Redhat |Redhat
+
+|Logging |Redhat |Shared |Shared |Shared |Red Hat
+
+|Application networking |Shared |Shared |Shared |Red Hat |Red Hat
+
+|Cluster networking |Redhat |Shared |Shared |Red Hat |Red Hat
+
+|Virtual networking |Shared |Shared |Shared |Shared |Shared
+
+|Master and infrastructure nodes |Redhat |Redhat |Redhat |Redhat |Redhat
```

Comment:
I'm not sure either, and since Vikram said no specific terminology decision has been made, I'd rather wait and see so that I only have to change these terms once.